ccd-definition/CaseEvent/User/UserEvents-SDO-nonprod.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "CREATE_SDO",
+    "Name": "Standard Direction Order",
+    "Description": "Standard Direction Order",
+    "DisplayOrder": 17,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N",
+    "EndButtonLabel": "Submit",
+    "RetriesTimeoutAboutToStartEvent": 0,
+    "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
```

Comment:
Don't think this second event is needed, I would have imagined that all the fields would be related to the `CREATE_SDO` event id